libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryWaterfallTests.java

```diff
@@ -172,7 +172,10 @@ public void ensureEndDialogCalled() {
         .startTest()
         .join();
 
-        Assert.assertEquals(7, counter[0]);
+        Mockito.verify(telemetryClient, Mockito.times(7)).trackEvent(
+            Mockito.anyString(),
+            Mockito.anyMap()
+        );
```

Comment:
Same

Reply:
Here I will left it as it was before

Reply:
Well now I remember why I did this, this line represents [this one](https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs#L139) from C#.